tests/test_service.py

```diff
@@ -90,4 +90,18 @@ def test_create_customer(self):
         # )
         # self.assertEqual(
         #     new_customer["available"], test_customer.available, "Availability does not match"
-        # )
\ No newline at end of file
+        # )
```

Comment:
These changes should be made in test_routes.py instead of test_service.py since we replaced test_service with test_routes in more recent changes